lib/fs-driver.js

```diff
@@ -371,3 +352,14 @@ FsDriver.prototype.removeWorkingDirectory = function (cb) {
 FsDriver.prototype.workingDiff = function (cb) {
   this.diff(this.root, this.working, cb);
 };
+
+/**
+ * Exposes the escape method.
+ */
+Object.defineProperty(FsDriver, 'escape', {
+  value: function (string) {
+    return new FsDriver('/').escape(string);
+  },
+  writable: false,
+  enumerable: true
+});
```

Comment:
I'm curious why expose `escape` like this, rather than making `escape` a class method (`FsDriver.escape`)